# # Bounding boxes topic here
# bounding_boxes_topic: "/sort_track/bounding_boxes"
# # Image topic here
# detect_image_topic: "/sort_track/image_track/compressed"

# show_image: false
# publish_image: true

# Bounding boxes topic here
# bounding_boxes_topic: "bounding_boxes_nms"
bounding_boxes_topic: "bounding_boxes_0"
# Image topic here
detect_image_topic: "output_image_nms"

camera_config_path: "/home/nvidia/ros_ws/src/traffic_count/config/config.yaml"
max_age: 5
min_hits: 1

show_image: true
publish_image: true